LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Protacon
```

Comment:
Should this be Protacon Group Ltd 🤔

Reply:
Hmm, I'll try to verify this.

Reply:
I verified this from Pirita, @ 2019 Protacon is fine